visiting.md

```diff
@@ -0,0 +1,101 @@
+
+_Please note, wearing a helmet while bicycling is the law in British Columbia and you can be fined ($29) for failing to do so. You can learn more about cycling in the province from [HUB Cycling](https://bikehub.ca/sites/default/files/imce/hub_cyclist_handbook_web.pdf)._
+
+{:.font-display}
+### Public transit
+
+Public transpit is operated by [Translink](https://www.translink.ca/) and the network consists of automated rapid transit called the Skytrain, Seabus ferries, commuter rail, and bus. Translink offers a [trip planner](https://www.translink.ca/trip-planner) and [route maps](https://www.translink.ca/schedules-and-maps) to aide passengers, and routes are searchable from most transit apps. 
+
+**Fares** are for the whole system and allow transfers between bus, Skytrain, and more on a single trip within 90 minutes. You can pay by cash, tap, and [Compass Card](https://www.compasscard.ca/). There are three [fare zones](https://www.translink.ca/transit-fares/pricing-and-fare-zones), with a single 1-ZONE trip costing $3.15 for cash or tap and $2.55 if you use the Compass card. 
+
+The airport is [Vancouver International airport (YVR)](https://www.yvr.ca/), from which you can travel to downtown Vancouver or across the Metro area via Skytrain along the [Canada Line](https://www.translink.ca/schedules-and-maps/skytrain). There is an [additional $5 fare](https://www.translink.ca/transit-fares/transferring-and-addfare) for taking the Skytrain from the airport.
+
+
+{:.font-display}
+
+## Staying in Vancouver
+
+For those looking at alternative accommodations including platforms like Airbnb, we wanted to provide context about the current housing market. 
+
+Vancouver is an expensive place to live and visit. Like many cities around the world, Vancouver is experiencing an [affordability crisis](https://globalnews.ca/news/10401449/vancouver-full-blown-crisis-housing-affordability-report/) compounded by a decades-long [housing crisis](https://thetyee.ca/Culture/2023/05/03/What-Behind-Housing-Crisis-Canada/), in part driven by housing that would otherwise be available to rentals being treated as speculative investments for the operation of [short-term rentals](https://www.cbc.ca/news/canada/british-columbia/short-term-rental-accomodation-act-bc-rules-and-enforcement-1.7177836). As a result, Vancouver rent is the [most expensive in Canada](https://vancouversun.com/news/local-news/vancouver-rent-flat-decline-most-expensive-report), an increase in [bad-faith evictions](https://www.theglobeandmail.com/real-estate/vancouver/article-landlord-use-now-most-frequently-used-tool-to-evict-bc-renters/), and [tenant unions](https://www.vancouvertenantsunion.ca/).
```

Comment:
```suggestion
Vancouver is an expensive place to live and visit. Like many cities around the world, Vancouver is experiencing an [affordability crisis](https://globalnews.ca/news/10401449/vancouver-full-blown-crisis-housing-affordability-report/) compounded by a decades-long [housing crisis](https://thetyee.ca/Culture/2023/05/03/What-Behind-Housing-Crisis-Canada/), in part driven by housing that would otherwise be available to rentals being treated as speculative investments for the operation of [short-term rentals](https://www.cbc.ca/news/canada/british-columbia/short-term-rental-accomodation-act-bc-rules-and-enforcement-1.7177836). As a result, Vancouver rent is the [most expensive in Canada](https://vancouversun.com/news/local-news/vancouver-rent-flat-decline-most-expensive-report) and there has been an increase in [bad-faith evictions](https://www.theglobeandmail.com/real-estate/vancouver/article-landlord-use-now-most-frequently-used-tool-to-evict-bc-renters/). The [Vancouver Tenants Union](https://www.vancouvertenantsunion.ca/about) has members who live in the 50% of the city that is rentals and collectively ensures their rights as tenants are respected.
```